<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>FileViewLayout</class>
  <widget class="QWidget" name="FileViewLayout">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>364</width>
        <height>639</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Form1</string>
    </property>
    <layout class="QVBoxLayout">
      <property name="margin">
        <number>0</number>
      </property>
      <property name="spacing">
        <number>0</number>
      </property>
      <item>
        <widget class="QTabWidget" name="m_pTabWidget">
          <property name="toolTip" stdset="0">
            <string/>
          </property>
          <widget class="QWidget" name="tab1">
            <attribute name="title">
              <string/>
            </attribute>
            <layout class="QVBoxLayout">
              <property name="margin">
                <number>0</number>
              </property>
              <property name="spacing">
                <number>0</number>
              </property>
              <item>
                <widget class="FileList" name="m_pFileList">
                  <property name="toolTip" stdset="0">
                    <string>Project File List</string>
                  </property>
                </widget>
              </item>
            </layout>
          </widget>
          <widget class="QWidget" name="tab2">
            <attribute name="title">
              <string/>
            </attribute>
            <layout class="QVBoxLayout">
              <property name="margin">
                <number>0</number>
              </property>
              <property name="spacing">
                <number>0</number>
              </property>
              <item>
                <widget class="KFileTreeView" name="m_pFileTree">
                  <property name="toolTip" stdset="0">
                    <string>File Tree</string>
                  </property>
                </widget>
              </item>
            </layout>
          </widget>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <customwidgets>
    <customwidget>
      <class>FileList</class>
      <extends>QWidget</extends>
      <header location="local">filelist.h</header>
      <sizehint>
        <width>-1</width>
        <height>-1</height>
      </sizehint>
      <container>0</container>
      <sizepolicy>
        <hordata>7</hordata>
        <verdata>7</verdata>
      </sizepolicy>
      <pixmap>image0</pixmap>
    </customwidget>
    <customwidget>
      <class>KFileTreeView</class>
      <extends>QWidget</extends>
      <header location="global">kfiletreeview.h</header>
      <sizehint>
        <width>-1</width>
        <height>-1</height>
      </sizehint>
      <container>0</container>
      <sizepolicy>
        <hordata>7</hordata>
        <verdata>7</verdata>
      </sizepolicy>
      <pixmap>image0</pixmap>
    </customwidget>
  </customwidgets>
  <images>
    <image name="image0">
      <data format="XBM.GZ" length="79">789c534e494dcbcc4b554829cdcdad8c2fcf4c29c95030e0524611cd48cd4ccf28010a1797249664262b2467241641a592324b8aa363156c15aab914146aadb90067111b1f</data>
    </image>
  </images>
  <includes>
      <include location="local">filelist.h</include>
  </includes>
</ui>
